modules/oe_content_tender/config/install/field.field.node.oe_tender.oe_teaser.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_teaser
+    - node.type.oe_tender
+  module:
+    - text
+id: node.oe_tender.oe_teaser
+field_name: oe_teaser
+entity_type: node
+bundle: oe_tender
+label: Teaser
+description: 'A short overview of the information on this page. The teaser will be displayed in list views and search engine results, not on the page itself. Limited to 150 characters for SEO purposes.'
+required: true
+translatable: true
```

Comment:
Field instances should always be not translatable by default, while storage fields should.

Reply:
fixed